Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModuleEditor.cs

```diff
@@ -2,14 +2,25 @@
 using System;
 using System.Linq;
 using UnityEditor;
+using UnityEditor.EventSystems;
+using UnityEngine.InputSystem.Editor;
 
 ////TODO: add button to automatically set up gamepad mouse cursor support
 
 namespace UnityEngine.InputSystem.UI.Editor
 {
     [CustomEditor(typeof(InputSystemUIInputModule))]
+    [InitializeOnLoad]
     internal class InputSystemUIInputModuleEditor : UnityEditor.Editor
     {
+        static InputSystemUIInputModuleEditor()
+        {
+#if UNITY_6000_0_OR_NEWER
```

Comment:
Indeed! Any reason to worry about 6000.0.0b15 and below not working with this version of the InputSystem package?

Reply:
Good point. So far I believe this kind of pattern is what we've been doing, which means we "don't care" about patch versions that don't have these changes. I believe we always assume the latest patch version of an Editor version because that's where the latest Input System package gets pulled cc @ekcoh .
I don't think for example: for a 6b1 version we would pull the latest 1.8.2 to be released for example. Or am I wrong? cc @stefanunity @Pauliusd01